packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -244,6 +244,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/webpack
 
 %changelog
+* Sat May 23 2020 Evgeni Golov - 3.16.0-0.13.rc1.1
+- Release rubygem-katello 3.16.0
```

Comment:
not crucial but this doesn't sound like the correct message

Reply:
It isn't. It never was, our tooling generates it incorrectly ;)

Reply:
(I have a todo somewhere to fix, but lazy ;))